ProcessLib/InitialCondition.cpp

```diff
@@ -21,7 +21,7 @@
 namespace ProcessLib
 {
 std::unique_ptr<InitialCondition> createUniformInitialCondition(
-    BaseLib::ConfigTree const& config)
+    BaseLib::ConfigTree const& config, int const tuple_size)
```

Comment:
It's not consistent now. (I'll check consistency related to this PR one more time.) 
I'd prefer tuple size here, so there is no ambiguity to number of components in a process...
MeshProperty uses tuple_size, same as VTK output.